doc/releases/changelog-dev.md

```diff
@@ -272,6 +272,12 @@ Here's a list of deprecations made this release. For a more detailed breakdown o
 
 <h3>Internal changes ⚙️</h3>
 
+* Enforce `optimize` module to be an auxiliary layer module.
+  [(#7418)](https://github.com/PennyLaneAI/pennylane/pull/7418)
+
+* The argument signatures for the `QNSPSAOptimizer`, `SPSAOptimizer`, and `ShotAdaptiveOptimizer` have been updated. While the first argument can still be provided positionally, all subsequent arguments are now keyword-only.
+  [(#7418)](https://github.com/PennyLaneAI/pennylane/pull/7418)
```

Comment:
I suppose this entry needs to go in a different category of the changelog? Breaking changes, even?